components/RuiPagination/readme.md

```diff
@@ -0,0 +1,178 @@
+---
+package: "@rhythm-ui/pagination"
+title: "rui-pagination"
+---
+
+# Pagination
+Pagination is used to allow the user to navigate between paginated content pages easily. 
+
+## Basic usage
+By default, pagination requires you to specify which page is the current page via the `current-page` attribute and how many pages are in the pagination in total via the `num-pages` attribute. When a pagination item is selected, the following 
+events will be fired: 
+
+`rui-pagination-next-click` when the next item is clicked
+`rui-pagination-prev-click` will be fired when the previous item is clicked
+`rui-pagination-item-click` will be fired when on of the pagination items is clicked, with the page number available via `event.detail.pageNumber`
+
+<Script script={
+    () => {
+        const el = document.getElementById('onclick-example');
+        if (el) {
+            el.addEventListener('rui-pagination-next-click', () => { alert('Next button clicked!')});
+
+            el.addEventListener('rui-pagination-prev-click', () => { alert('Previous button clicked!')});
+
+            el.addEventListener('rui-pagination-item-click', (e) => { alert(`Page ${e.detail.pageNumber} clicked!`)});
+        }
+    }
+}></Script>
+```html preview
+<script>
+    const el = document.getElementById('onclick-example');
+    if (el) {
+        el.addEventListener('rui-pagination-next-click', () => { alert('Next button clicked!')});
```

Comment:
You raise a good point, I have refactored it in the latest commits to only use one event. Because it is no longer just an 'item click' event I have renamed it to 'page request'.